client/js/components/document/DpMapSettingsPreview.vue

```diff
@@ -387,7 +387,8 @@ export default {
 
     isNotEmptyFeatureCollection (string) {
       try {
-        return JSON.parse(string).features.length > 0
+        const parsedFeatures = JSON.parse(string).features
+        return Array.isArray(parsedFeatures) ? parsedFeatures.length > 0 : Object.keys(JSON.parse(string).features).length > 0
```

Comment:
sorry, was not clean
```suggestion
        const parsedFeatures = JSON.parse(string).features

        return Array.isArray(parsedFeatures) ? parsedFeatures.length > 0 : Object.keys(parsedFeatures).length > 0
```